Nighttime-Operations/Observatory/DIMM/ops.rst

```diff
@@ -0,0 +1,195 @@
+
+
+From a terminal in the LSST WAN network, connect to the DIMM VM:
+
+.. code-block:: bash
+
+        (base) you@Ios ~ % ssh dimm@dimm.cp.lsst.org
+
+The password is located on the Operators 1Password Vault.
+ 
+
+Connect via telnet to the ``tt-meteo``. 
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  telnet 127.0.0.1 16301
+
+
+The prompt within OpenTPL, a plain-text client-server protocol, is ``AUTH OK 0 0``. Commands are prepended with a freely chosen command id (unsigned integer).
+More on this communication protocol can be found on the :ref:`DIMM general specs <DIMM-DIMM>`.
+
+The following two commands can be written once the prompt appears. 
+
+The first command will set the corrected sky temperature; the second will fix the sky to clear, confirming both observations are possible and safe. 
+
+.. code-block:: bash
+    
+        1 set sky.temp=-30
+    
+        2 set sky.status=0
+
+Quit telnet, pressing simultaneously ``Ctrl + ]``, write ``quit`` and click Enter. 
+
+The image below shows how the interaction with the OpenTPL client looks like and the two commands required to manually configure `tt-meteo`.
+
+.. figure:: ./_static/tt-meteo.png
+    :name: tt-meteo prerequisite
+
+    Prerequisite: Weather data
+
+Now the DIMM is ready to start operations. 
+
+Follow only one of the procedures below, either start the DIMM up in :ref:`automatic mode <Dimm_StartUp-Automatic>` or in 
+:ref:`manual mode <Dimm_StartUp-Manual>`
+
+.. _Dimm_StartUp-Automatic: 
+
+Starting Up the DIMM in Automatic Mode
+======================================
+
+Make sure you have run and read the :ref:`prerequisites <Dimm_StartUp-Prerequisites>` regarding the weather data. 
+
+
+In this mode of operations, the DIMM `ameba` will automatically select the targets from the star catalog. 
+
+To start an automatic DIMM observation, connect to `tt-master` where you will set the variable ``ameba.mode`` to 1:
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  telnet 127.0.0.1 16500
+
+
+and within this OpenTPL:
+
+.. code-block:: bash
+   
+        AUTH OK 0 0
+        1 set ameba.mode=1
+        
+
+Verify the DIMM CSC is enabled and data is appearing in the EFD.
+
+Once this mode is enabled, the DIMM will continue taking data until sunrise, and will run again the following night. 
+
+To manually stop the DIMM, see :ref:`Stop the DIMM <Dimm-Shutdown>`
+
+To monitor the status of the DIMM program, see :ref:`monitor the DIMM <Dimm-Monitor>`.
+
+.. _Dimm_StartUp-Manual:
+
+Starting up the DIMM in Manual Mode
+====================================
+
+Make sure you have run and read the :ref:`prerequisites <Dimm_StartUp-Prerequisites>` regarding the weather data. 
+
+In this mode of operations, the user needs to choose the target and run the ``monitor_dimm_2.py`` python script available in the dimm home directory:
+
+.. note:: Keep `mag` and `color` in 0.0.  
+
+.. code-block:: bash
+
+        dimm@dimm:~$  python3 monitor_dimm_2.py manual --name {target name} 
```

Comment:
please verify spectral type is an actual input!

Reply:
👍